### PR TITLE
AB#803 Feature - Toasts component

### PR DESCRIPTION
**Pull request by iths-jonathan-engman**
**Describe pull-request**  
- New feature - toasts component

**How to test**  
- cd sdds
- npm start

**Screenshots**  
image

### CODE REVIEW

File: theme/light/src/components/_toasts.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:root,
+html {
+  --sdds-toasts-background-color: #{get-colour(grey-900)};
```
Comment (helloimela) [inline theme/light/src/components/_toasts.scss:3]: Use prefix for the "sdds". See example in "sdds/theme/light/src/components/_textfield.scss"

**Review by helloimela — CHANGES_REQUESTED** [review]

File: theme/light/src/components/_toasts.scss (hunk 1 above)
Comment (helloimela) [reply theme/light/src/components/_toasts.scss:3]: --#{$prefix}-

**Review by helloimela — CHANGES_REQUESTED** [review]
Can you rename all files from toasts to toast ?

**Comment by iths-jonathan-engman** [comment]
> Can you rename all files from toasts to toast ?

Noticed that tooltip(s) need the same intervention

**Review by helloimela — APPROVED** [review]
looks good!

**Review by mmexvr — APPROVED** [review]

**Review by vichtertz — CHANGES_REQUESTED** [review]
Awesome work! I only have 2 comments:
- The cross should be grey-50 not white (it's my bad, I did it wrong in the design)
- When there is only a header, it looks like there is more space on the bottom than on the top. The header container should be 16px but there is extra space under (cf. screenshot), is it possible to fix it? 

Screenshot 2021-05-18 at 11 34 37 (2)

**Review by vichtertz — CHANGES_REQUESTED** [review]
The UI is perfect well done! Just one last thing, could you change these two namings in the Controls:
- Headline should be Header
- Subheadline should be Subheader

Screenshot 2021-05-19 at 11 40 00

**Comment by helloimela** [comment]
image

**The styling is a bit messed up in the website preview. You need to add "float:left" to the toast-headline span**
See image below

image

**Comment by helloimela** [comment]
image

The link is affected by the website, you need to add class "sdds-toast-link" to the "a" element. And it is semantically wrong to have "a" inside "span". Change the structure to:

```
<div class="sdds-toast-body">
   <a href="#"  class="sdds-toast-link">Link example</a>
</div>
```

**Pull request merged**